tests/components/denonavr/test_media_player.py

```diff
@@ -0,0 +1,39 @@
+"""The tests for the denonavr media player platform."""
+from unittest.mock import MagicMock
+
+from homeassistant.components import media_player
+from homeassistant.components.demo import media_player as demo
+import homeassistant.components.denonavr as denonavr
+from homeassistant.components.denonavr import ATTR_COMMAND, DOMAIN, SERVICE_GET_COMMAND
+from homeassistant.const import ATTR_ENTITY_ID
+from homeassistant.setup import async_setup_component
+
+
+async def test_setup_demo_platform(hass):
+    """Test setup."""
+    mock = MagicMock()
+    add_entities = mock.MagicMock()
+    await demo.async_setup_platform(hass, {}, add_entities)
+    assert add_entities.call_count == 1
+
+
+async def test_get_command(hass):
+    """Test generic command functionality."""
+
+    await hass.async_add_executor_job(denonavr.setup, hass, {})
```

Comment:
Use `async_setup_component` to set up the media_player component and the denonavr platform. The denonavr component should be set up automatically as a dependency of the denonavr media_player platform.